Tests/Rules/UseToExportFieldsInManifest.tests.ps1

```diff
@@ -83,7 +83,7 @@ Describe "UseManifestExportFields" {
             $results[0].Extent.Text | Should -Be "'*'"
         }
 
-        It "suggests corrections for AliasesToExport with wildcard" -pending:($IsLinux -or $IsMacOS) {
+        It "suggests corrections for AliasesToExport with wildcard" -pending:($IsCoreClr) {
```

Comment:
is this tracked by an issue?

Reply:
No but Pester shows the number of pending tests. We should maybe create an issue specific for the non-Windows builds and fix/investigate it as part of making the non-Windows builds. As you can see in the other Linux PR, there are at the moment 13 failing tests on Linux.

Reply:
I created an issue [here](https://github.com/PowerShell/PSScriptAnalyzer/issues/944)